changelog.md

```diff
@@ -2,3 +2,8 @@
 
 ## 0.1.0
 - Initial release
+
+## Unreleased
+- Added MockModule
```

Comment:
Put class names between backticks

Reply:
Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/4/commits/ce756b192e37b7c6e076754d55ea7b2f0e72124b